strings/src/main/res/values/strings.xml

```diff
@@ -7,22 +7,22 @@
         <item quantity="other">Delete %d photos</item>
     </plurals>
     <plurals name="delete_photos_confirmation">
-        <item quantity="one">Are you sure you want to delete this %d photo?</item>
-        <item quantity="other">Are you sure you want to delete these %d photos?</item>
+        <item quantity="one">Delete this photo?</item>
+        <item quantity="other">Delete these %d photos?</item>
     </plurals>
-    <string name="error_refreshing_people">There was an error refreshing people</string>
-    <string name="error_loading_photo_details">There was an error loading photo details</string>
-    <string name="error_searching">There was an error while searching</string>
-    <string name="feed_sync_freq">Full photo feed sync frequency</string>
-    <string name="feed_sync_freq_changed">Feed sync frequency changed</string>
-    <string name="feed_sync_network_changed">Feed sync network requirement changed</string>
-    <string name="feed_sync_charging_changed">Feed sync charging requirements changed</string>
+    <string name="error_refreshing_people">Could not refresh people</string>
+    <string name="error_loading_photo_details">Could not load photo details</string>
+    <string name="error_searching">Could not carry out search</string>
+    <string name="feed_sync_freq">Sync the feed every …</string>
+    <string name="feed_sync_freq_changed">Now syncing everything every %1$d</string>
```

Comment:
Added a placeholder here.
Makes more sense to say what the actual cadence is I think.